VarSECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+
+Tell them where to go, how often they can expect to get an update on a
```

Comment:
def report_vulnerability(email, description, steps, system_info):
    """
    Function to report a vulnerability in the software.
    :param email: Email address of the reporter.
    :param description: Detailed description of the vulnerability.
    :param steps: Steps to reproduce the vulnerability.
    :param system_info: Information about the affected system(s) and software version(s).
    :return: None
    """
    print("Thank you for reporting the vulnerability. We will get back to you within 24 hours.")
    print("You can expect to receive updates on the status of the vulnerability report every 3 days.")
    print("If the vulnerability is accepted, we will work to fix it as quickly as possible and release a security update.")
    print("If the vulnerability is declined, we will provide an explanation for why it was not considered a security risk.")
    # send an email to security team with the vulnerability information
    send_email(to_email='security@example.com', subject='Vulnerability Report', body=f'Email: {email}\n Description: {description} \n Steps: {steps} \n System Info: {system_info